web/src/beta/components/TabButton/index.tsx

```diff
@@ -0,0 +1,38 @@
+import type { FC } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+export type Props = {
+  label: string;
+  onClick?: () => void;
+  selected?: boolean;
+};
+
+const TabButton: FC<Props> = ({ label, onClick, selected }) => {
+  return (
+    <Button onClick={onClick} disabled={selected}>
+      {label}
+    </Button>
+  );
+};
+
+type ButtonProps = {
+  disabled?: boolean;
+};
+
+const Button = styled.button<ButtonProps>`
+  background: ${({ disabled }) => (disabled ? "#232226" : "inherit")};
+  padding: 8px 12px;
+  height: 35px;
+  border-radius: 4px;
+  :hover {
+    background: #232226;
+    transition: all 0.5s ease;
+  }
+  color: ${({ disabled }) => (disabled ? "#C7C5C5" : "#4A4A4A")};
```

Comment:
Could you use theme colors?
You can directly access the theme via `${({ theme }) => theme.editorNavBar.xxx}` in styled component.

Current beta design regered as `darkTheme`.
1. search color code from [here](https://github.com/reearth/reearth/blob/main/web/src/services/theme/colors.ts)
2. search a variable that the color(result of 1) is used from [here](https://github.com/reearth/reearth/blob/main/web/src/services/theme/darkTheme.ts). You can define new fields if needed.

Please check colors you used again :pray: